src/main/java/net/minecraftforge/client/event/FOVModifierEvent.java

```diff
@@ -24,35 +24,35 @@
 import net.minecraft.util.math.MathHelper;
 import net.minecraftforge.eventbus.api.Event;
 
-public class FOVUpdateEvent extends Event
+public class FOVModifierEvent extends Event
 {
     private final PlayerEntity entity;
     private final float fov;
     private float newfov;
 
-    public FOVUpdateEvent(PlayerEntity entity, float fov)
+    public FOVModifierEvent(PlayerEntity entity, float fov)
     {
         this.entity = entity;
         this.fov = fov;
-        this.setNewfov(MathHelper.lerp(Minecraft.getInstance().gameSettings.field_243227_aN, 1.0F, fov));
+        this.setFOV(MathHelper.lerp(Minecraft.getInstance().gameSettings.field_243227_aN, 1.0F, fov));
     }
 
     public PlayerEntity getEntity()
     {
         return entity;
     }
 
-    public float getFov()
+    public float getOriginalFOV()
     {
         return fov;
     }
 
-    public float getNewfov()
+    public float getFOV()
     {
         return newfov;
     }
 
-    public void setNewfov(float newfov)
+    public void setFOV(float newfov)
```

Comment:
Might want to rename the parameter, too.

Reply:
My intention with making this PR is to do as much of the breaking changes needed, and only breaking changes as much as possible. A rename of a method parameter does not affect the method's signature and (to my understanding) does not break binary compatibility, and therefore can be done in a later PR outside of the breaking changes window. _(My intention is to have this PR to be merged as soon as possible within the breaking changes window, and another PR such as my events javadocs PR can rename the internal details, such as field and method parameter names)_
But, to be consistent with fixing all of the issues flagged by review, I will fix this.